owasp-top10-2017-apps/a1/mongection/README.md

```diff
@@ -7,7 +7,11 @@ Mongection is a simple NodeJS web application that simulates a login page. It ha
 
 ## Index
 
+<<<<<<< HEAD
```

Comment:
Ops, it looks like these were some conflicts you were trying to resolve? 😅 Would you mind fixing them, please?